packages/oas/test/lib/openapi-to-json-schema.test.ts

```diff
@@ -741,25 +737,19 @@ describe('`format` support', () => {
       ['integer', 'uint64', 0, 2 ** 64 - 1], // 0 to 1844674407370955161
       ['number', 'float', 0 - 2 ** 128, 2 ** 128 - 1], // -3.402823669209385e+38 to 3.402823669209385e+38
       ['number', 'double', 0 - Number.MAX_VALUE, Number.MAX_VALUE],
-    ])('`type: %s`', (type, format, min, max) => {
+    ])('`type: %s`', (type, format) => {
```

Comment:
probably want remove this entire test since it was testing adding the min/maxes

Reply:
done!